cluster/images/etcd/migrate/rollback_v2.go

```diff
@@ -314,12 +314,33 @@ func toTTLOptions(r *pb.Request) store.TTLOptionSet {
 }
 
 func applyRequest(r *pb.Request, applyV2 etcdserver.ApplierV2) {
+	// TODO: find a sane way to perform this cast or avoid it in the first place
+	reqV2 := &etcdserver.RequestV2{
```

Comment:
This seems orthogonal to the main purpose of this PR which is level up the deps to 3.3

Reply:
the etcd2-specific tools are being removed in https://github.com/kubernetes/kubernetes/pull/69577, fyi

Reply:
Unfortunately, This change is required by the version bump, due to a change in the etcd codebase.  Look forward to stripping out the etcd2 code in 1.14

Reply:
> Generally LGTM, but I would split out your v2 compat changes into it's own PR, unless this update somehow forces the issue. 

^ seems to apply then.